recipes/10/10_over.md

```diff
@@ -0,0 +1,53 @@
+# 10 Rolling Aggregations on Time Series Data
+
+:bulb: This example will show how to aggregate multiple rows, which are related to the current row, via an `OVER` window. A typical use case are rolling aggregations.
+
+The source table (`temperature_measurements`) is backed by the [`faker` connector](https://github.com/knaufk/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+OVER window aggregates compute an aggregated value for every input row over a range of ordered rows. 
+In contrast to GROUP BY aggregates, OVER aggregates do not reduce the number of result rows to a single row for every group. 
+Instead, OVER aggregates produce an aggregated value for every input row.
+
+The order needs to be defined by a [time attribute](https://docs.ververica.com/user_guide/sql_development/table_view.html#time-attributes). 
+The range of rows can be defined by a number of rows or a time interval. 
+
+In this example, we are trying to identify outliers in the `temperature_measurements` table. 
+For this, we use an `OVER` window to calculate, for each measurement, the maximum, minimum, standard deviation and average temperature across all measurements for the same city over the previous minute. 
+You can imagine that a subsequent query would filter out any temperature measurement that are higher or lower than the average by more than four standard deviations.
```

Comment:
Formulate as a quoted part, saying "As an exercise, you can write another query to ..."?